branding/css/susemanager-theme.less

```diff
@@ -1303,6 +1303,9 @@ select.small-select {
   text-overflow: ellipsis;
 }
 
+.panel-default.no-padding {
```

Comment:
Since this is a utility that seems to recur here and below, perhaps we could extract it into a standalone utility class?

```less
.no-padding {
  padding: 0;
}
```

Reply:
I fully agree. Only, few lines below there is `.panel-heading.no-padding`, wouldn't this be part of a wider refactoring? Otherwise the latter will be hided forever

Reply:
Anyway, just thinking twice, I can refactor it now instead of leaving for later, and drop the `panel-heading` one as well.